src/features/isanteplus/findPatient.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Patient Search
+
+  Background:
+    Given User logs in the system
+
+  @findPatient
+  Scenario: Searching for a patient
+  And User clicks on Find Patient App
+  And User enters missing patient
+  Then Search Page returns no patients
+  And  User enters moses mutesa
```

Comment:
you dont need to hard code the name 
can you use the cucmber example tables to load the necessary data??

Reply:
Resolved thanks